hw/ip/adc_ctrl/doc/_index.md

```diff
@@ -139,7 +139,7 @@ The list below describes how the counters interpret the filter results:
     This will cause the block to raise an interrupt if it was not already doing so.
   * If a filter is a hit and is also enabled in {{< regref "adc_wakeup_ctl" >}} the corresponding filter generates a wakeup.
   * Note that the debounce counter will remain at its threshold until the set of filters are changed by software to debounce a different event or if the current match changes.
-    *  This implies that a stable matching event continuously matches until some condition in the system (changed filter settings or changed ADC output) alters the result.
+    *  This implies that a stable matching event continuously matches until some condition in the system (changed filter settings, changed ADC output, software issued fsm reset in {{< regref "adc_fsm_rst" >}}) alters the result.
```

Comment:
Should we also add a line to the programmer's guide further below (e.g. at the end of the `Use Case` section)  saying that this needs to be reset after a match? Just to make it explicit...

Reply:
so that's the thing, this doesn't actually need to be done.
It's now how it was intended to be used. The main way this was supposed to be done is that you simply change the filter settings for the "next" match you're looking for.

I asked the same question in the issue.

Reply:
Ah I see. Maybe the test where this came up just reuses the same filter setting?

Reply:
yeah i guess so...